artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -4062,6 +4079,20 @@
                </xsd:simpleType>
             </xsd:element>
 
+            <xsd:element name="page-full-policy" maxOccurs="1" minOccurs="0">
+               <xsd:annotation>
+                  <xsd:documentation>
+                     After entering page mode, a second limit will be set by page-limit-bytes, page-limit-messages. page-full-policy will configure what to do when that limit is reach.
```

Comment:
page-limit-bytes and/or page-limit-messages. The page-full-policy...

...when that limit is reach**ed.**